packages/visualizations/app/styles/navi-visualizations/components/number-format-selector.less

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2017, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+.number-format-selector {
+    &__format-list {
+        display: flex;
+        flex-direction: column;
+    }
+    
+    &__format {
+        height: @report-form-height;
+    }
+
+    &__doc-link {
+      font-size: 15px;
+    }
+    
+    &__format-input {
+        font-size: @font-size-base-large;
+    }
+    
+    &__section {
+        .config-section();
+    }
+    
+    &__section-header {
+        .metric-label-config__section-header
+    }
+    
+    &__text-input {
+        .metric-label-config__text-input
```

Comment:
Not sure sure. Maybe we can tweak this when working on table config

Reply:
So since this is the generic component that would be reused, i feel its not right to include the metric label styles here. Can we remove this here and style this specifically for the metric-label in the `metric-label.less`?

Reply:
Can we address this comment please?